### ISSUE TITLE
[BUG] Create thumbnail button breaks after switching items

### ISSUE THREAD

**Issue by masaball** [avalonmediasystem/avalon#5407]
**Describe the bug**
On a media object with multiple items, the create thumbnail button works on initial page load, but generates a console error "player.player is null" after a new item is selected from the structured navigation.

**To Reproduce**
Steps to reproduce the behavior, including the results:
1. Go to https://avalon-dev.dlib.indiana.edu/media_objects/g158bh28p
2. Click on 'Create Thumbnail'
3. See that the modal pops up and populates
4. Close modal
5. Select a new item from the structured navigation
6. Click on 'Create Thumbnail'
7. Modal does not pop up
8. Check browser console to see error

**Expected behavior**
Modal pops up and functions properly after switching items, no error is logged to the console

**Screenshots/Screen Recording**
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMzBSQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4836160c12f0311722ad1faa4fd2ad86f7583d76/Screenshot_20231006_092603.png)

**Environment (please complete the following information):**
 - Laptop
 - OS: Ubuntu 22.04
 - Browser: Firefox 117
- [x] avalon-dev
- [ ] avalon-staging
- [ ] mco-staging
- [ ] mco-production
- [ ] demo

**Release:**
7.7

**Comment by elynema** [comment]
When you first load the page, the JS for the modal loads with details of first section. When you switch sections, there is no code to update the JS for the modal to the data for new section. So this is likely causing the issue.

**Comment by elynema** [comment]
Share button does have some functionality to make it work whenever the canvas/section is switched. There is an event it's listening for to get the latest info.

**Comment by joncameron** [comment]
Setting the thumbnail is working for me for the first section, but switching to any other section and launching the Create Thumbnail modal only shows a broken image icon. The video below is a bit janky but I was able to reproduce this and get this in the modal:
<img width="780" alt="Screenshot 2023-11-03 at 7 45 38 AM" src="https://github.com/avalonmediasystem/avalon/assets/1406618/57e0d3af-49e1-4741-9636-7289ab8288df">

Console error:
`
GET
https://avalon-dev.dlib.indiana.edu/master_files/6d56zw601/poster?offset=73.541672&preview=true
[HTTP/2 500  800ms]`



https://github.com/avalonmediasystem/avalon/assets/1406618/08e12519-29a9-4546-aac4-83bad4bef3aa

**Comment by joncameron** [comment]
Is this a separate bug where the poster offset image from the stream can't be pulled properly?

**Comment by masaball** [comment]
Yes, this seems to be a different bug. The information for pulling the poster image is populated correctly in the request, but the `extract_frame` method is bombing. Here's the server log:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEtlQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2f9e4cc2e5f119de0bf4f8669fd304505a188d47/Screenshot_20231103_092229.png)

**Comment by joncameron** [comment]
@joncameron to close this and create a new bug issue for this frame extraction error.

**Comment by joncameron** [comment]
Created #5455 for followup on the frame extraction error.
